Taskfile.yml

```diff
@@ -0,0 +1,163 @@
+# https://taskfile.dev/
+
+version: '3'
+
+vars:
+  BINARY_PATH: "${HOME}/.terraform.d/plugins/registry.terraform.io/opslevel/opslevel"
+  OPSLEVEL_DIR: "{{.ROOT_DIR}}/opslevel"
+  VERSION_DYNAMIC:
+    sh: echo "$(git describe --tags --abbrev=0 | tr -d 'v')$(date +%s)"
+  WORKSPACE_DIR: "{{.ROOT_DIR}}/workspace"
+
+tasks:
+
+  install-changie:
+    desc: Install changie
+    aliases: [get-changie, get-changie]
+    cmds:
+      - echo "Installing changie..."
+      - task: install-changie-mac-os
```

Comment:
you can actually make this work for everyone by doing `go install github.com/miniscruff/changie@latest`
It just requires that they have setup `GOBIN` correctly

Reply:
My plan here is to just run `go install github.com/miniscruff/changie@latest` as is. go install [handled things a bit differently](https://pkg.go.dev/cmd/go#hdr-Compile_and_install_packages_and_dependencies) before go 1.20 but since [go 1.19 EOL was last week](https://endoflife.date/go) I think it's fair to make some assumptions.